src/core/app-template.cc

```diff
@@ -191,10 +200,15 @@ app_template::run_deprecated(int ac, char ** av, std::function<void ()>&& func)
                     .positional(_pos_opts)
                     .run()
             , configuration);
+        program_options::variables_map_extracting_visitor visitor(configuration);
+        _opts.config_file.mutate(visitor);
         _conf_reader(configuration);
     } catch (bpo::error& e) {
         fmt::print("error: {}\n\nTry --help.\n", e.what());
         return 2;
+    } catch (...) {
+        fmt::print("error: {}\n\n", std::current_exception());
```

Comment:
better off sending error messages to stderr, like
```c++
fmt::print(stderr, "error: {}\n\n", std::current_exception());
```

but i understand, probably we should change all of them in another change, as we are writing error messages to `stdout` in app template already.

Reply:
Yes, that's separate patch. Here I made all error logging consistent